evm/src/executor/inspector/cheatcodes/util.rs

```diff
@@ -222,7 +248,23 @@ pub fn apply<DB: Database>(
 ) -> Option<Result> {
     Some(match call {
         HEVMCalls::Addr(inner) => addr(inner.0),
-        HEVMCalls::Sign(inner) => sign(inner.0, inner.1.into(), data.env.cfg.chain_id.into()),
+        // [function sign(uint256,bytes32)] Used to sign bytes32 digests using the given private key
+        HEVMCalls::Sign0(inner) => sign(inner.0, inner.1.into(), data.env.cfg.chain_id.into()),
+        // [function createWallet(string)] Used to derive private key and label the wallet with the
+        // same string
+        HEVMCalls::CreateWallet0(inner) => {
+            create_wallet(U256::from(keccak256(&inner.0)), Some(inner.0.clone()), state)
+        }
+        // [function createWallet(uint256)] creates a new wallet with the given private key
+        HEVMCalls::CreateWallet1(inner) => create_wallet(inner.0, None, state),
```

Comment:
@Evalir, I looked around for an existing doc schema for each of these `match call` arms, but I couldn't find any so I made my own. Let me know what you think 👍